code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -553,7 +553,7 @@ GLOBAL_VAR(bomb_set)
 	var/turf/bomb_location = get_turf(src)
 	var/area/A = get_area(src)
 	if( bomb_location && is_station_level(bomb_location.z) )
-		if( (bomb_location.x < (128 - NUKERANGE)) || (bomb_location.x > (128 + NUKERANGE)) || (bomb_location.y < (128 - NUKERANGE)) || (bomb_location.y > (128 + NUKERANGE)) && (!(A in GLOB.the_station_areas)))
+		if( (bomb_location.x < (128 - NUKERANGE)) || (bomb_location.x > (128 + NUKERANGE)) || (bomb_location.y < (128 - NUKERANGE)) || (bomb_location.y > (128 + NUKERANGE)) && (!(A in SSmapping.existing_station_areas)))
```

Comment:
This may conflict/be an alternate solution to #21518, I haven't taken a close look but I know this is the same code that is changed there.